### DISCUSSION TITLE
International stream sync issue

### DISCUSSION THREAD

**Discussion by VictorColomb** [f1multiviewer/issue-tracker#26] [General]
The driver tracker and data channel views are usually perfectly synced. However, the International stream is 6 or 7 seconds behind, even after clicking on "Live" at the bottom-right of the view or after pressing `l`.

Maybe add an option to manually adjust delays between the different windows?

**Comment by jeppeolesen** [comment]
Hey @VictorColomb - closing this discussion as stale. If it's still an issue, please submit it over here: https://github.com/f1multiviewer/issue-tracker/issues
